src/reactable.jsx

```diff
@@ -344,6 +344,7 @@
             var Ths = [];
             for (var index = 0; index < this.props.columns.length; index++) {
                 var column = this.props.columns[index];
+                var thClass = column.key
```

Comment:
Nice :+1: Should we improve anything else? And how about the test too, should I add more tests to any other sections?